Orso.Arpa.Application/Orso.Arpa.Application.csproj

```diff
@@ -9,4 +9,5 @@
     <ProjectReference Include="..\Orso.Arpa.Infrastructure\Orso.Arpa.Infrastructure.csproj" />
     <ProjectReference Include="..\Orso.Arpa.Persistence\Orso.Arpa.Persistence.csproj" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+
```

Comment:
wozu die Leerzeile?

Reply:
ups, is wech

Reply:
da war vorher fälschlicherweise das ical package drinne